packages/cli/src/commands/start.ts

```diff
@@ -58,17 +57,23 @@ interface InstallDependenciesOptions {
 }
 
 /** dependencies for the project */
-function installServicesAsDependencies({ packageManager, projectAbsolutePath, services }: InstallDependenciesOptions): void {
+async function installServicesAsDependencies({ packageManager, projectAbsolutePath, services }: InstallDependenciesOptions): Promise<void> {
   // in case service is empty we add dummy package. we use typescript for sure, so we use it.
   const pkg = ([...services.map(service => serviceToConnector(service)), "typescript"]).join(" ");
-  const command = installCommand({ packageManager, pkg });
+  const commands = installCommand({ packageManager, pkg });
+  const [command, ...args] = commands.split(" ");
+
+  const s = p.spinner();
 
-  console.log("📦 Package installation in progress...");
+  s.start("📦 Package installation in progress...");
 
-  execSync(command, {
-    cwd: projectAbsolutePath,
-    stdio: "inherit",
+  await x(command, args, {
```

Comment:
we can use child_process library
if we want async exec, we warp the callback base exec function with promise

or....
Is there any difference?

Reply:
idk, but it was easier.
oh wait, we just promisify child_process!!! I'll try it!!

Reply:
Oops.. typo...
Thank you for understanding so well.

Reply:
@sunrabbit123 I modified with promisify and update mock function! please review it!

Reply:
we reduced 10kb to remove tinyexec!